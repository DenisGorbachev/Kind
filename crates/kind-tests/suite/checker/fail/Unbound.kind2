Tew : U60
Tew = owo