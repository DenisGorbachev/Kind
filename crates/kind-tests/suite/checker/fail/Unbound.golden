   ERROR  Cannot find the definition 'owo'.

      /--[suite/checker/fail/Unbound.kind2:2:7]
      |
    1 |    Tew : Data.U60
    2 |    Tew = owo
      |          v--
      |          \Here!

      Hint: Take a look at the rules for name searching at https://github.com/Kindelia/Kind2/blob/master/guide/naming.md

